### ISSUE TITLE
Phantom Train

### ISSUE THREAD

**Issue by TheBuffalax** [gameboy9/FF6KefkaRush#34]
i dont know if you want to leave the ghost vendors in or not

**Comment by gameboy9** [comment]
Yeah, I'm planning on removing those, now that I got the item distribution of monster drops improved.

**Comment by TheBuffalax** [comment]
The monster drop improvement is very nice.
